packages/resolver/lib/sources/fs.ts

```diff
@@ -57,29 +57,31 @@ export class FS implements ResolverSource {
     return path.basename(sourcePath, ".sol");
   }
 
-  async resolve(importPath: string, importedFrom: string) {
-    importedFrom = importedFrom || "";
-    const possiblePaths = [
-      importPath,
-      path.join(path.dirname(importedFrom), importPath)
-    ];
+  async resolve(importPath: string, importedFrom?: string) {
```

Comment:
Is `importedFrom` still necessary?

Reply:
Well, remember, resolver sources have to maintain a uniform interface.  You're right that `importedFrom` is ignored here though... probably it should get marked with a `_` (husky didn't remind me of this since its not working!).  It's possible you could drop it without causing compilation problems?  I didn't try it.

Reply:
Good point about the uniform interface. In that case the `_` param prefix  would be appropriate. However, lets defer lint issues until husky is working.

Reply:
Eh, I'd rather change this now rather than make some future person deal with it.  I'll go change this now and then merge once tests pass again.